src/Faker/Provider/zh_CN/Person.php

```diff
@@ -5,45 +5,73 @@
 class Person extends \Faker\Provider\Person
 {
     protected static $maleNameFormats = array(
-        '{{firstNameMale}}{{lastName}}',
+        '{{lastName}}{{firstNameMale}}',
     );
 
     protected static $femaleNameFormats = array(
-        '{{firstNameFemale}}{{lastName}}',
+        '{{lastName}}{{firstNameFemale}}',
     );
 
-    protected static $firstNameMale = array(
-        '任', '彭', '杨', '林', '毛', '胡', '陈', '雷', '龙',
-        // below are unknown gender firstnames
-        '丁', '万', '严', '于', '何', '余', '侯', '傅', '冯', '刘', '卢', '史', '叶',
-        '吕', '吴', '周', '唐', '夏', '姚', '姜', '孔', '孙', '宋', '崔', '廖',
-        '张', '徐', '方', '曹', '曾', '朱', '李', '杜', '梁', '武', '段', '江',
-        '汪', '沈', '洪', '潘', '熊', '王', '田', '白', '秦', '程', '罗', '苏',
-        '范', '莫', '萧', '董', '蒋', '蔡', '薛', '袁', '覃', '许', '谢', '谭',
-        '贺', '贾', '赖', '赵', '邓', '邱', '邵', '邹', '郝', '郭', '金', '钟',
-        '钱', '阎', '陆', '陶', '韦', '韩', '顾', '马', '高', '魏', '黄', '黎',
-        '龚',
+    protected static $lastName = array(
+        // According to http://baike.baidu.com/view/6109935.htm,
```

Comment:
Please use the phpDoc format (starting with `/*`) and move it one line up, before the array initialization.